tests/test_shell.py

```diff
@@ -199,14 +323,233 @@ def test_remove(self):
         assert job_to_remove in project
         assert job_to_remove.doc.a == 0
         assert len(job_to_remove.doc) == 1
-        with pytest.deprecated_call():
-            self.call('python -m signac rm --clear {}'.format(job_to_remove.get_id()).split())
+        self.call('python -m signac rm --clear {}'.format(job_to_remove.id).split())
         assert job_to_remove in project
         assert len(job_to_remove.doc) == 0
-        with pytest.deprecated_call():
-            self.call('python -m signac rm {}'.format(job_to_remove.get_id()).split())
+        self.call('python -m signac -v rm {}'.format(job_to_remove.id).split())
+        assert job_to_remove not in project
+        with pytest.raises(ExitCodeError):
```

Comment:
I think some comment (here and in similar locations elsewhere) as to why certain calls are expected to fail would be very helpful.